src/components/ContentPreview/ContentPreview.js

```diff
@@ -1055,10 +1059,13 @@ class ContentPreview extends PureComponent<Props, State> {
                     {hasHeader && (
                         <Header
                             file={file}
+                            token={token}
                             onClose={onClose}
                             onPrint={this.print}
                             canDownload={this.canDownload()}
                             onDownload={this.download}
+                            showOpenWithButton={showOpenWithButton}
```

Comment:
Can this be folded into `openWithProps`, like `openWithProps.show`?

Reply:
my only issue with that is that `show` would never actually be applied to the element. So we would want to remove it when spreading the props on the component?